cmd/soroban-cli/src/commands/contract/init.rs

```diff
@@ -219,16 +206,11 @@ impl Runner {
             "target",
             "Cargo.lock",
         ];
-        for entry in read_dir(from).map_err(|e| {
-            self.print
-                .errorln(format!("Error reading directory: {from:?}"));
-            e
-        })? {
-            let entry = entry.map_err(|e| {
-                self.print
-                    .errorln(format!("Error reading entry in directory: {from:?}"));
-                e
-            })?;
+        for entry in read_dir(from)
+            .map_err(|e| Error::Io(format!("Error reading directory: {from:?}"), e))?
```

Comment:
Error strings usually start with a lower case letter I think. We might have some inconsistency, but I think that's what I saw last I checked.

Reply:
Also we can drop the prefix "Error " because when outputted errors are proceeded with "error: " iirc.

Reply:
Good call to both of those - I'll make an update!